docs/api-docs/cart-and-checkout/custom-order-confirmation.md

```diff
@@ -61,6 +61,11 @@ You can upload a custom order confirmation page to your store's server using Web
 
 ## Installing a custom order confirmation page  
 
+There are two ways to install a custom checkout on a store.
```

Comment:
```suggestion
You can install a custom checkout on a store in the following two ways:
```
this revision ^^ is one solution to make the following bullet points less wordy. you can also make them anchor links to the sections below.